dockerfiles/Dockerfile.cross

```diff
@@ -1,5 +1,5 @@
 
-FROM    golang:1.8.1
+FROM    golang:1.8.3
```

Comment:
I see the [Dockerfile.dev](https://github.com/docker/cli/blob/b7b7d784d8007aea0d8cff54c67d19e960cb4639/dockerfiles/Dockerfile.dev#L2) floats the golang version to latest patch of 1.8. I'd suggest floating the version in `Dockerfile.cross` as well so they match, .e.g. `FROM golang:1.8`